### PR TITLE
ft successful registration jwt 184638229

### PR DESCRIPTION
**Pull request by gershomlapaix**
This pr is about the registration of users in our system
The task was to send tokens on successful registration; it means that there must be the registration first.

* To test this task is to use the endpoint: `localhost:3000/api/v1/auth/register` using post verb
* The basic fields are firstName, lastName, email, password, and passwordConfirm

Note: you must first do migration and push the `users` table in your local db. Here are the commands:
- npx prisma migrate dev
- npx prisma db push

It is associated with pivotal tracker #184638229 story

### CODE REVIEW

**Review by iradukundairenee — COMMENTED** [review]
is this chore? @gershomlapaix

File: src/controllers/auth.controller.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      console.log("impossibility");
+    }
+
+    // const { access_token, refresh_token } = await signTokens(user);
+    // res.cookie("access_token", access_token, accessTokenCookieOptions);
+    // res.cookie("refresh_token", refresh_token, refreshTokenCookieOptions);
+    // res.cookie("logged_in", true, {
+    //   ...accessTokenCookieOptions,
+    //   httpOnly: false,
+    // });
```
Comment (iradukundairenee) [inline src/controllers/auth.controller.ts]: remove this comment

File: src/controllers/auth.controller.ts [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+    res.status(201).json({
+      status: "success",
+      data: {
+        user,
+        // access_token,
+        // refresh_token,
```
Comment (iradukundairenee) [inline src/controllers/auth.controller.ts]: also remove this comment

**Comment by Daniel-Dukundane** [comment]
Kindly fix the error in the github actions.

**Review by iradukundairenee — APPROVED** [review]

**Pull request merged**